### PR TITLE
update spring poms to use unreleased eventhubs and servicebus

### PR DESCRIPTION
**Pull request by anuchandy**

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-actuator/pom.xml [hunk 1]
```diff
@@ -71,8 +71,8 @@
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-messaging-eventhubs</artifactId>
-      <version>5.18.7</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;dependency} -->
+      <artifactId>azure-messaging-eventhubs</artifactId> <!-- {x-version-update;unreleased_com.azure:azure-messaging-eventhubs;dependency} -->
+      <version>5.19.0-beta.3</version>
```
Comment (alzimmermsft) [inline sdk/spring/spring-cloud-azure-actuator/pom.xml:75]: Should be one line

**Review by alzimmermsft — APPROVED** [review]

**Comment by anuchandy** [comment]
Overriding the check enforcer, see the linked issue to enable unreleased on spring compact pipeline.

**Pull request merged**